Sources/Networking/IntroEligibilityResponse.swift

```diff
@@ -16,7 +16,7 @@ import Foundation
 // All parameters that are required to process the reponse from a GetIntroEligibility API call.
 struct IntroEligibilityResponse {
 
-    let result: Result<HTTPResponse, Error>
+    let result: Result<HTTPResponse<[String: Any]>, Error>
```

Comment:
The new implementation still allows extracting a plain `[String: Any]` to maintain compatibility for types that aren't `Decodable` yet.